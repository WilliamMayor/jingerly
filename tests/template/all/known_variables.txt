In this file there is a variable known={{ known }} that should be replaced.