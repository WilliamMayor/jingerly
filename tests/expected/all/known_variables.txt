In this file there is a variable known=this is known that should be replaced.